src/containers/Dashboard/reducer.js

```diff
@@ -0,0 +1,32 @@
+import { fromJS } from 'immutable';
+
+import { EMPTY_ARRAY } from '../../constants/app';
+import appConstants from './constants';
+
+const initialState = fromJS({
+  cntProducts: 3,
+  randomProducts: EMPTY_ARRAY,
+  loading: false,
+  error: false,
+});
+
+function appReducer(state = initialState, action) {
+  switch (action.type) {
+    case appConstants.UPDATE_RANDOM_BANNER_PRODUCTS:
+      return state.set('randomProducts', fromJS(action.randomProducts));
+
+    case appConstants.SET_ERROR:
+      return state.set('error', action.error);
+
+    case appConstants.SET_LOADING:
+      return state.set('loading', action.loading);
+
+    case appConstants.SET_CNT_PRODUCTS:
```

Comment:
Thus, this is not really needed. In your saga you will make them random and according to number in action.